src/Image/Image.php

```diff
@@ -32,6 +32,30 @@ class Image extends File
      */
     protected $dimensions;
 
+    /**
+     * @var array
+     */
+    public static $resizable = [
```

Comment:
Idea for a more speaking name that can also avoid conflicts later if we ever need to have instance properties with the same name:

```suggestion
    public static $resizableTypes = [
```